modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/common/enums/prmPageRoute.ts

```diff
@@ -12,4 +12,5 @@
 export enum PRMPageRoute {
 	CREATE_MDF_REQUEST = 'marketing/mdf-requests/new',
 	MDF_REQUESTS_LISTING = 'marketing/mdf-requests',
+	MDF_CLAIM_LISTING = 'marketing/mdf-claim',
```

Comment:
```suggestion
	MDF_CLAIM_LISTING = 'marketing/mdf-claims',
```

nao deveria ser no plural?